server/webhooks/failing_policies_test.go

```diff
@@ -265,84 +411,3 @@ func TestSendBatchedPOSTs(t *testing.T) {
 		})
 	}
 }
-
-func TestFilterPolicies(t *testing.T) {
```

Comment:
Any way we can still test this kind of thing?

Reply:
It is kind of tested here https://github.com/fleetdm/fleet/blob/f11d13f3d1f40183ed9b3d1f29473df1b8310a3b/server/webhooks/failing_policies_test.go#L204 because the test includes a failing policy that should be filtered out when the webhook gets triggered ie policy id 2 is not in failing policies webhook policy ids. Do you think we should add separate tests for filtering? I could also make a similar change to `TestTriggerFailingPoliciesWebhookBasic` to add a failing policy that gets filtered out.